deployments/helm/nsm/templates/_helpers.yaml

```diff
@@ -0,0 +1,18 @@
+---
+{{/* Configure Jaeger environment */}}
+{{- define "jaeger.env" -}}
+{{- if .Values.global.JaegerTracing }}
+- name: TRACER_ENABLED
+  value: "true"
+- name: JAEGER_AGENT_HOST
+  value: "jaeger.nsm-system"
+- name: JAEGER_AGENT_PORT
+  value: "6831"
+{{- end }}
+{{- end -}}
+
+{{/* Configure INSECURE environment variable */}}
+{{- define "insecure.env" }}
+- name: INSECURE
+  value: {{ if .Values.insecure }}"true"{{ else }}"false"{{ end }}
```

Comment:
Does `{{ .Values.insecure |  quote }}` works?

Reply:
Yes, with addition of 'default' since 'make' overrides default values in .tpl with empty values from command line.
thank you!